### DISCUSSION TITLE
Ideal weight settings for Kolors IP-adapter Face ID?

### DISCUSSION THREAD

**Discussion by KobayakawaSaeSama** [cubiq/ComfyUI_IPAdapter_plus#693] [General]
Would you guys discuss or share the ideal weight settings for Kolors (both weight,weight_faceidv2 and weight_Kolors) here? Don't know if the weight settings provided in [https://github.com/cubiq/ComfyUI_IPAdapter_plus/blob/main/examples/IPAdapter_FaceIDv2_Kolors.json](url) is tested to be optimal . Thanks!

**Comment by cubiq** [comment]
I need to check the code for Kolors because I believe something is not right yet... but now everything is about flux so didn't have much time

**Reply by KobayakawaSaeSama** [reply]
Fair enough, thanks
